SOURCES/etc/xapi.d/plugins/xcpngutils/pid/__init__.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from .. import timeout
```

Comment:
I don't see this file in the commit, and when trying to use the plugin, I got this error:
```
stderr: Traceback (most recent call last):
  File "/etc/xapi.d/plugins/raid.py", line 7, in <module>
    from xcpngutils import configure_logging, run_command, pidfile
  File "/etc/xapi.d/plugins/xcpngutils/__init__.py", line 12, in <module>
    from xcpngutils.pid import PidFile
  File "/etc/xapi.d/plugins/xcpngutils/pid/__init__.py", line 3, in <module>
    from .. import timeout

```

Reply:
The file is in the commit:
https://github.com/xcp-ng/xcp-ng-xapi-plugins/blob/a5fb58c0073d925450f7a481e664a32e4fcbbfb2/SOURCES/etc/xapi.d/plugins/xcpngutils/__init__.py (It's the new name of xcpngutils.py because the pidfile module is a part of the xcpngutils module/folder now.)

Maybe you have the xcpngutils.py file in your system and python uses it instead of the xcpngutils folder?

Reply:
Ah! I see, thanks.